R/environment.r

```diff
@@ -44,3 +44,23 @@ init_shadowenv <- function() {
         stop("'edit()' not yet supported in the Jupyter R kernel")
     })
 }
+
+
+init_cran_repo <- function() {
+    local({
```

Comment:
if you say so. :-) I thought so, too, but RStudio can't make so many things wrong... but this seems to be in since the beginning, so removing it... done